frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx

```diff
@@ -127,17 +127,18 @@ const RenderResendModal = (props: {
                     <Button
                         type="button"
                         size="small"
-                        disabled={props.loading}
-                        onClick={() => props.startResend()}
+                        outline
+                        onClick={props.closeResendModal}
                     >
-                        Trigger Resend
+                        Cancel
                     </Button>
                     <Button
                         type="button"
                         size="small"
-                        onClick={props.closeResendModal}
+                        disabled={props.loading}
+                        onClick={() => props.startResend()}
                     >
-                        Cancel
+                        Trigger Resend
```

Comment:
This diff is confusing, but I just swapped the position of the Cancel button to be consistent with other places.